aggregator.go

```diff
@@ -36,10 +35,12 @@ func (a *Aggregator) Put(data []byte, partitionKey string) {
 	// later, we will add shard-mapper same as the KPL use.
 	// see: https://github.com/a8m/kinesis-producer/issues/1
 	if len(a.pkeys) == 0 {
-		a.pkeys = append(a.pkeys, partitionKey)
+		a.pkeys = []string{partitionKey}
 		a.nbytes += len([]byte(partitionKey))
 	}
 	keyIndex := uint64(len(a.pkeys) - 1)
+
+	a.nbytes += 8
```

Comment:
Let's move this into a const at the top level declaration and use it in the producer.go as well.